pypeit/spectrographs/keck_deimos.py

```diff
@@ -218,6 +207,9 @@ def default_pypeit_par(cls):
         par['scienceframe']['process']['sigclip'] = 4.0
         par['scienceframe']['process']['objlim'] = 1.5
 
+        # Find objects
+        par['reduce']['findobj']['find_fwhm'] = 10.  # 1.1"
```

Comment:
Just checking that, given recent chatter on Slack, is this still what we want to do?

Reply:
Can you please add a comment here indicating the default (unbanned pixel scale) and hence how many arc seconds this corresponds to?

Reply:
done